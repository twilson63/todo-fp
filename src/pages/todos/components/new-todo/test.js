/* global test, expect */
import React from 'react'
import { shallow } from 'enzyme'
import NewTodo from './'

test('NewTodo Component renders correctly', done => {
  const props = {}
  const result = shallow(
    <NewTodo {...props} />
  )

  expect(result).toMatchSnapshot()
  done()
})
