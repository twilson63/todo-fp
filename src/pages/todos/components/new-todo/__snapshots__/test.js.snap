exports[`test NewTodo Component renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <withHandlers(NewTodo)
    todo=""
    updateTodo={[Function]} />,
  "nodes": Array [
    <withHandlers(NewTodo)
      todo=""
      updateTodo={[Function]} />,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <withState(withHandlers(NewTodo)) />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": _class2 {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": Object {
          "stateValue": "",
        },
        "updateStateValue": [Function],
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <withHandlers(NewTodo)
          todo=""
          updateTodo={[Function]} />,
        "_debugID": 2,
        "_renderedOutput": <withHandlers(NewTodo)
          todo=""
          updateTodo={[Function]} />,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <withState(withHandlers(NewTodo)) />,
}
`;
